cmd/emp/releases.go

```diff
@@ -226,7 +227,36 @@ func runRollback(cmd *Command, args []string) {
 		os.Exit(2)
 	}
 	ver := strings.TrimPrefix(args[0], "v")
+	rollbackSafetyCheck(ver, appname)
+
 	rel, err := client.ReleaseRollback(appname, ver, message)
 	must(err)
 	log.Printf("Rolled back %s to v%s as v%d.\n", appname, ver, rel.Version)
 }
+
+func rollbackSafetyCheck(verStr, appname string) {
+	// Grab head release to get the current version
+	hrels, err := client.ReleaseList(appname, &heroku.ListRange{
+		Field:      "version",
+		Max:        1,
+		Descending: true,
+	})
+	must(err)
+
+	currVer := hrels[0].Version
+	verNum, err := strconv.Atoi(verStr)
+	must(err)
+
+	diff := currVer - verNum
+	if diff >= 10 {
+		var vconfirmation string
+		fmt.Printf("WARNING. Attempting to rollback %d versions from v%d to v%d.\n", diff, currVer, verNum)
+		fmt.Printf("To proceed, type v%d: ", verNum)
+		fmt.Scanln(&vconfirmation)
```

Comment:
You could probably re-use the [mustConfirm](https://github.com/remind101/empire/blob/66a68e1a8c5b9f861e319d373da34396d379ffa0/cmd/emp/util.go#L136) helper here.